controllers/profiles/common/persistence/postgresql.go

```diff
@@ -0,0 +1,203 @@
+// Copyright 2023 Apache Software Foundation (ASF)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package persistence
+
+import (
+	"fmt"
+	"sync"
+
+	corev1 "k8s.io/api/core/v1"
+
+	operatorapi "github.com/apache/incubator-kie-kogito-serverless-operator/api/v1alpha08"
+	"github.com/apache/incubator-kie-kogito-serverless-operator/controllers/profiles/common/constants"
+)
+
+const (
+	defaultSchemaName   = "default"
+	defaultDatabaseName = "sonataflow"
+
+	timeoutSeconds       = 3
+	failureThreshold     = 5
+	initialPeriodSeconds = 15
+	initialDelaySeconds  = 10
+	successThreshold     = 1
+
+	postgreSQLCPULimit      = "500m"
+	postgreSQLMemoryLimit   = "256Mi"
+	postgreSQLMemoryRequest = "256Mi"
+	postgreSQLCPURequest    = "100m"
+
+	defaultPostgreSQLUsername  = "sonataflow"
+	defaultPostgresSQLPassword = "sonataflow"
+)
+
+var (
+	WorkflowConfig *syncConfig
```

Comment:
Global var where we store the persistence information for a specific platform, I have asked already about how this works. Happy to learn if there's a go trick I'm missing.